notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>Do we use column *names* or *labels* more often?</p><p> </p><p>I typically prefer *labels*, since "names" can conflict with Index.name / MultiIndex.names.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>ok with using labels. I'll start listing these type of _conventions_ in order to make them consistent in the following tutorials.</p><p> </p>